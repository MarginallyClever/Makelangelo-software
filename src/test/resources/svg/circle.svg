<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 24.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 595.28 841.89" style="enable-background:new 0 0 595.28 841.89;" xml:space="preserve">
<style type="text/css">
	.st0{fill:#FFFFFF;stroke:#000000;stroke-miterlimit:10;}
</style>
<circle class="st0" cx="255.86" cy="273.11" r="164.04"/>
</svg>
