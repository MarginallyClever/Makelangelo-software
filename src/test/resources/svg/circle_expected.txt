[TOOL R0 G0 B0 A255 D1.000, TRAVEL X0.645 Y-0.324, DRAW_LINE X0.645 Y-0.328, DRAW_LINE X0.645 Y-0.331, DRAW_LINE X0.645 Y-0.335, DRAW_LINE X0.645 Y-0.338, DRAW_LINE X0.644 Y-0.341, DRAW_LINE X0.644 Y-0.345, DRAW_LINE X0.644 Y-0.348, DRAW_LINE X0.643 Y-0.352, DRAW_LINE X0.643 Y-0.355, DRAW_LINE X0.642 Y-0.358, DRAW_LINE X0.642 Y-0.362, DRAW_LINE X0.641 Y-0.365, DRAW_LINE X0.640 Y-0.368, DRAW_LINE X0.639 Y-0.372, DRAW_LINE X0.639 Y-0.375, DRAW_LINE X0.638 Y-0.378, DRAW_LINE X0.637 Y-0.381, DRAW_LINE X0.636 Y-0.385, DRAW_LINE X0.635 Y-0.388, DRAW_LINE X0.633 Y-0.391, DRAW_LINE X0.632 Y-0.394, DRAW_LINE X0.631 Y-0.397, DRAW_LINE X0.630 Y-0.401, DRAW_LINE X0.628 Y-0.404, DRAW_LINE X0.627 Y-0.407, DRAW_LINE X0.626 Y-0.410, DRAW_LINE X0.624 Y-0.413, DRAW_LINE X0.622 Y-0.416, DRAW_LINE X0.621 Y-0.419, DRAW_LINE X0.619 Y-0.422, DRAW_LINE X0.617 Y-0.425, DRAW_LINE X0.616 Y-0.428, DRAW_LINE X0.614 Y-0.431, DRAW_LINE X0.612 Y-0.433, DRAW_LINE X0.610 Y-0.436, DRAW_LINE X0.608 Y-0.439, DRAW_LINE X0.606 Y-0.442, DRAW_LINE X0.604 Y-0.444, DRAW_LINE X0.602 Y-0.447, DRAW_LINE X0.600 Y-0.450, DRAW_LINE X0.597 Y-0.452, DRAW_LINE X0.595 Y-0.455, DRAW_LINE X0.593 Y-0.457, DRAW_LINE X0.591 Y-0.460, DRAW_LINE X0.588 Y-0.462, DRAW_LINE X0.586 Y-0.465, DRAW_LINE X0.583 Y-0.467, DRAW_LINE X0.581 Y-0.469, DRAW_LINE X0.578 Y-0.471, DRAW_LINE X0.576 Y-0.474, DRAW_LINE X0.573 Y-0.476, DRAW_LINE X0.570 Y-0.478, DRAW_LINE X0.568 Y-0.480, DRAW_LINE X0.565 Y-0.482, DRAW_LINE X0.562 Y-0.484, DRAW_LINE X0.559 Y-0.486, DRAW_LINE X0.556 Y-0.488, DRAW_LINE X0.554 Y-0.490, DRAW_LINE X0.551 Y-0.491, DRAW_LINE X0.548 Y-0.493, DRAW_LINE X0.545 Y-0.495, DRAW_LINE X0.542 Y-0.496, DRAW_LINE X0.539 Y-0.498, DRAW_LINE X0.536 Y-0.500, DRAW_LINE X0.533 Y-0.501, DRAW_LINE X0.530 Y-0.502, DRAW_LINE X0.527 Y-0.504, DRAW_LINE X0.523 Y-0.505, DRAW_LINE X0.520 Y-0.506, DRAW_LINE X0.517 Y-0.507, DRAW_LINE X0.514 Y-0.509, DRAW_LINE X0.511 Y-0.510, DRAW_LINE X0.507 Y-0.511, DRAW_LINE X0.504 Y-0.512, DRAW_LINE X0.501 Y-0.513, DRAW_LINE X0.498 Y-0.513, DRAW_LINE X0.494 Y-0.514, DRAW_LINE X0.491 Y-0.515, DRAW_LINE X0.488 Y-0.516, DRAW_LINE X0.484 Y-0.516, DRAW_LINE X0.481 Y-0.517, DRAW_LINE X0.477 Y-0.517, DRAW_LINE X0.474 Y-0.518, DRAW_LINE X0.471 Y-0.518, DRAW_LINE X0.467 Y-0.519, DRAW_LINE X0.464 Y-0.519, DRAW_LINE X0.461 Y-0.519, DRAW_LINE X0.457 Y-0.519, DRAW_LINE X0.454 Y-0.519, DRAW_LINE X0.450 Y-0.519, DRAW_LINE X0.447 Y-0.519, DRAW_LINE X0.444 Y-0.519, DRAW_LINE X0.440 Y-0.519, DRAW_LINE X0.437 Y-0.519, DRAW_LINE X0.433 Y-0.519, DRAW_LINE X0.430 Y-0.518, DRAW_LINE X0.427 Y-0.518, DRAW_LINE X0.423 Y-0.517, DRAW_LINE X0.420 Y-0.517, DRAW_LINE X0.417 Y-0.516, DRAW_LINE X0.413 Y-0.516, DRAW_LINE X0.410 Y-0.515, DRAW_LINE X0.407 Y-0.514, DRAW_LINE X0.403 Y-0.513, DRAW_LINE X0.400 Y-0.513, DRAW_LINE X0.397 Y-0.512, DRAW_LINE X0.393 Y-0.511, DRAW_LINE X0.390 Y-0.510, DRAW_LINE X0.387 Y-0.509, DRAW_LINE X0.384 Y-0.507, DRAW_LINE X0.381 Y-0.506, DRAW_LINE X0.377 Y-0.505, DRAW_LINE X0.374 Y-0.504, DRAW_LINE X0.371 Y-0.502, DRAW_LINE X0.368 Y-0.501, DRAW_LINE X0.365 Y-0.500, DRAW_LINE X0.362 Y-0.498, DRAW_LINE X0.359 Y-0.496, DRAW_LINE X0.356 Y-0.495, DRAW_LINE X0.353 Y-0.493, DRAW_LINE X0.350 Y-0.491, DRAW_LINE X0.347 Y-0.490, DRAW_LINE X0.344 Y-0.488, DRAW_LINE X0.341 Y-0.486, DRAW_LINE X0.339 Y-0.484, DRAW_LINE X0.336 Y-0.482, DRAW_LINE X0.333 Y-0.480, DRAW_LINE X0.330 Y-0.478, DRAW_LINE X0.328 Y-0.476, DRAW_LINE X0.325 Y-0.474, DRAW_LINE X0.323 Y-0.471, DRAW_LINE X0.320 Y-0.469, DRAW_LINE X0.317 Y-0.467, DRAW_LINE X0.315 Y-0.465, DRAW_LINE X0.313 Y-0.462, DRAW_LINE X0.310 Y-0.460, DRAW_LINE X0.308 Y-0.457, DRAW_LINE X0.306 Y-0.455, DRAW_LINE X0.303 Y-0.452, DRAW_LINE X0.301 Y-0.450, DRAW_LINE X0.299 Y-0.447, DRAW_LINE X0.297 Y-0.444, DRAW_LINE X0.295 Y-0.442, DRAW_LINE X0.293 Y-0.439, DRAW_LINE X0.291 Y-0.436, DRAW_LINE X0.289 Y-0.433, DRAW_LINE X0.287 Y-0.431, DRAW_LINE X0.285 Y-0.428, DRAW_LINE X0.283 Y-0.425, DRAW_LINE X0.282 Y-0.422, DRAW_LINE X0.280 Y-0.419, DRAW_LINE X0.278 Y-0.416, DRAW_LINE X0.277 Y-0.413, DRAW_LINE X0.275 Y-0.410, DRAW_LINE X0.274 Y-0.407, DRAW_LINE X0.272 Y-0.404, DRAW_LINE X0.271 Y-0.401, DRAW_LINE X0.270 Y-0.397, DRAW_LINE X0.268 Y-0.394, DRAW_LINE X0.267 Y-0.391, DRAW_LINE X0.266 Y-0.388, DRAW_LINE X0.265 Y-0.385, DRAW_LINE X0.264 Y-0.381, DRAW_LINE X0.263 Y-0.378, DRAW_LINE X0.262 Y-0.375, DRAW_LINE X0.261 Y-0.372, DRAW_LINE X0.261 Y-0.368, DRAW_LINE X0.260 Y-0.365, DRAW_LINE X0.259 Y-0.362, DRAW_LINE X0.258 Y-0.358, DRAW_LINE X0.258 Y-0.355, DRAW_LINE X0.257 Y-0.352, DRAW_LINE X0.257 Y-0.348, DRAW_LINE X0.257 Y-0.345, DRAW_LINE X0.256 Y-0.341, DRAW_LINE X0.256 Y-0.338, DRAW_LINE X0.256 Y-0.335, DRAW_LINE X0.256 Y-0.331, DRAW_LINE X0.256 Y-0.328, DRAW_LINE X0.256 Y-0.324, DRAW_LINE X0.256 Y-0.321, DRAW_LINE X0.256 Y-0.318, DRAW_LINE X0.256 Y-0.314, DRAW_LINE X0.256 Y-0.311, DRAW_LINE X0.256 Y-0.307, DRAW_LINE X0.257 Y-0.304, DRAW_LINE X0.257 Y-0.301, DRAW_LINE X0.257 Y-0.297, DRAW_LINE X0.258 Y-0.294, DRAW_LINE X0.258 Y-0.291, DRAW_LINE X0.259 Y-0.287, DRAW_LINE X0.260 Y-0.284, DRAW_LINE X0.261 Y-0.281, DRAW_LINE X0.261 Y-0.277, DRAW_LINE X0.262 Y-0.274, DRAW_LINE X0.263 Y-0.271, DRAW_LINE X0.264 Y-0.267, DRAW_LINE X0.265 Y-0.264, DRAW_LINE X0.266 Y-0.261, DRAW_LINE X0.267 Y-0.258, DRAW_LINE X0.268 Y-0.255, DRAW_LINE X0.270 Y-0.251, DRAW_LINE X0.271 Y-0.248, DRAW_LINE X0.272 Y-0.245, DRAW_LINE X0.274 Y-0.242, DRAW_LINE X0.275 Y-0.239, DRAW_LINE X0.277 Y-0.236, DRAW_LINE X0.278 Y-0.233, DRAW_LINE X0.280 Y-0.230, DRAW_LINE X0.282 Y-0.227, DRAW_LINE X0.283 Y-0.224, DRAW_LINE X0.285 Y-0.221, DRAW_LINE X0.287 Y-0.218, DRAW_LINE X0.289 Y-0.215, DRAW_LINE X0.291 Y-0.213, DRAW_LINE X0.293 Y-0.210, DRAW_LINE X0.295 Y-0.207, DRAW_LINE X0.297 Y-0.204, DRAW_LINE X0.299 Y-0.202, DRAW_LINE X0.301 Y-0.199, DRAW_LINE X0.303 Y-0.197, DRAW_LINE X0.306 Y-0.194, DRAW_LINE X0.308 Y-0.192, DRAW_LINE X0.310 Y-0.189, DRAW_LINE X0.313 Y-0.187, DRAW_LINE X0.315 Y-0.184, DRAW_LINE X0.317 Y-0.182, DRAW_LINE X0.320 Y-0.180, DRAW_LINE X0.323 Y-0.177, DRAW_LINE X0.325 Y-0.175, DRAW_LINE X0.328 Y-0.173, DRAW_LINE X0.330 Y-0.171, DRAW_LINE X0.333 Y-0.169, DRAW_LINE X0.336 Y-0.167, DRAW_LINE X0.339 Y-0.165, DRAW_LINE X0.341 Y-0.163, DRAW_LINE X0.344 Y-0.161, DRAW_LINE X0.347 Y-0.159, DRAW_LINE X0.350 Y-0.157, DRAW_LINE X0.353 Y-0.156, DRAW_LINE X0.356 Y-0.154, DRAW_LINE X0.359 Y-0.152, DRAW_LINE X0.362 Y-0.151, DRAW_LINE X0.365 Y-0.149, DRAW_LINE X0.368 Y-0.148, DRAW_LINE X0.371 Y-0.146, DRAW_LINE X0.374 Y-0.145, DRAW_LINE X0.377 Y-0.144, DRAW_LINE X0.381 Y-0.142, DRAW_LINE X0.384 Y-0.141, DRAW_LINE X0.387 Y-0.140, DRAW_LINE X0.390 Y-0.139, DRAW_LINE X0.393 Y-0.138, DRAW_LINE X0.397 Y-0.137, DRAW_LINE X0.400 Y-0.136, DRAW_LINE X0.403 Y-0.135, DRAW_LINE X0.407 Y-0.135, DRAW_LINE X0.410 Y-0.134, DRAW_LINE X0.413 Y-0.133, DRAW_LINE X0.417 Y-0.133, DRAW_LINE X0.420 Y-0.132, DRAW_LINE X0.423 Y-0.131, DRAW_LINE X0.427 Y-0.131, DRAW_LINE X0.430 Y-0.131, DRAW_LINE X0.433 Y-0.130, DRAW_LINE X0.437 Y-0.130, DRAW_LINE X0.440 Y-0.130, DRAW_LINE X0.444 Y-0.130, DRAW_LINE X0.447 Y-0.130, DRAW_LINE X0.450 Y-0.130, DRAW_LINE X0.454 Y-0.130, DRAW_LINE X0.457 Y-0.130, DRAW_LINE X0.461 Y-0.130, DRAW_LINE X0.464 Y-0.130, DRAW_LINE X0.467 Y-0.130, DRAW_LINE X0.471 Y-0.131, DRAW_LINE X0.474 Y-0.131, DRAW_LINE X0.477 Y-0.131, DRAW_LINE X0.481 Y-0.132, DRAW_LINE X0.484 Y-0.133, DRAW_LINE X0.488 Y-0.133, DRAW_LINE X0.491 Y-0.134, DRAW_LINE X0.494 Y-0.135, DRAW_LINE X0.498 Y-0.135, DRAW_LINE X0.501 Y-0.136, DRAW_LINE X0.504 Y-0.137, DRAW_LINE X0.507 Y-0.138, DRAW_LINE X0.511 Y-0.139, DRAW_LINE X0.514 Y-0.140, DRAW_LINE X0.517 Y-0.141, DRAW_LINE X0.520 Y-0.142, DRAW_LINE X0.523 Y-0.144, DRAW_LINE X0.527 Y-0.145, DRAW_LINE X0.530 Y-0.146, DRAW_LINE X0.533 Y-0.148, DRAW_LINE X0.536 Y-0.149, DRAW_LINE X0.539 Y-0.151, DRAW_LINE X0.542 Y-0.152, DRAW_LINE X0.545 Y-0.154, DRAW_LINE X0.548 Y-0.156, DRAW_LINE X0.551 Y-0.157, DRAW_LINE X0.554 Y-0.159, DRAW_LINE X0.556 Y-0.161, DRAW_LINE X0.559 Y-0.163, DRAW_LINE X0.562 Y-0.165, DRAW_LINE X0.565 Y-0.167, DRAW_LINE X0.568 Y-0.169, DRAW_LINE X0.570 Y-0.171, DRAW_LINE X0.573 Y-0.173, DRAW_LINE X0.576 Y-0.175, DRAW_LINE X0.578 Y-0.177, DRAW_LINE X0.581 Y-0.180, DRAW_LINE X0.583 Y-0.182, DRAW_LINE X0.586 Y-0.184, DRAW_LINE X0.588 Y-0.187, DRAW_LINE X0.591 Y-0.189, DRAW_LINE X0.593 Y-0.192, DRAW_LINE X0.595 Y-0.194, DRAW_LINE X0.597 Y-0.197, DRAW_LINE X0.600 Y-0.199, DRAW_LINE X0.602 Y-0.202, DRAW_LINE X0.604 Y-0.204, DRAW_LINE X0.606 Y-0.207, DRAW_LINE X0.608 Y-0.210, DRAW_LINE X0.610 Y-0.213, DRAW_LINE X0.612 Y-0.215, DRAW_LINE X0.614 Y-0.218, DRAW_LINE X0.616 Y-0.221, DRAW_LINE X0.617 Y-0.224, DRAW_LINE X0.619 Y-0.227, DRAW_LINE X0.621 Y-0.230, DRAW_LINE X0.622 Y-0.233, DRAW_LINE X0.624 Y-0.236, DRAW_LINE X0.626 Y-0.239, DRAW_LINE X0.627 Y-0.242, DRAW_LINE X0.628 Y-0.245, DRAW_LINE X0.630 Y-0.248, DRAW_LINE X0.631 Y-0.251, DRAW_LINE X0.632 Y-0.255, DRAW_LINE X0.633 Y-0.258, DRAW_LINE X0.635 Y-0.261, DRAW_LINE X0.636 Y-0.264, DRAW_LINE X0.637 Y-0.267, DRAW_LINE X0.638 Y-0.271, DRAW_LINE X0.639 Y-0.274, DRAW_LINE X0.639 Y-0.277, DRAW_LINE X0.640 Y-0.281, DRAW_LINE X0.641 Y-0.284, DRAW_LINE X0.642 Y-0.287, DRAW_LINE X0.642 Y-0.291, DRAW_LINE X0.643 Y-0.294, DRAW_LINE X0.643 Y-0.297, DRAW_LINE X0.644 Y-0.301, DRAW_LINE X0.644 Y-0.304, DRAW_LINE X0.644 Y-0.307, DRAW_LINE X0.645 Y-0.311, DRAW_LINE X0.645 Y-0.314, DRAW_LINE X0.645 Y-0.318, DRAW_LINE X0.645 Y-0.321, DRAW_LINE X0.645 Y-0.324]